### DISCUSSION TITLE
cypress open-ct doesn't show the user intercace, only Cannot Get /__/

### DISCUSSION THREAD

**Discussion by chkrause** [cypress-io/cypress#16286] [Component Testing]
Hi,
I installed component testing as described on [cypress.io.](https://docs.cypress.io/guides/component-testing/introduction#What-is-Component-Testing)

Cypress run-ct command is working fine and all component test run. But when I'm trying to start cypress open-rt it's starting chrome but without any useful content. Only Cannot Get /__/. 

Do you have any ideas why?

![image](https://user-images.githubusercontent.com/40870406/116691137-af994200-a9ba-11eb-88fb-9144ab4ea52a.png)

**Comment by lmiller1990** [comment]
Hi! Can you provide some more info? What OS? Version of Cypress? etc. Thanks!

**Reply by chkrause** [reply]
ElevateBart already provided a repo for testing this behaviour on my machine. 
(Windows 10 Enterprise 10.0.18363
npm 7.6.2
yarn 1.22.5
Node v15.9.0)
With this https://github.com/elevatebart/cy-ct-cra repo I got the same result as in my project.

![image](https://user-images.githubusercontent.com/40870406/117408268-f84b8080-af0f-11eb-96ea-e621f2b827b9.png)
